### ISSUE TITLE
Look up zRIF's also in VitaSmith license.db database

### ISSUE THREAD

**Issue by mmozeiko** [mmozeiko/pkgi#23]
When items in pkgi.txt have no zRIF strings, try to lookup fake license in @VitaSmith license.db file.

**Comment by MarioMasta64** [comment]
but whos to confirm that its right? or what the hash is? if there is no zRif i dont think there would be a hash, nonetheless this sounds coolio.

**Comment by MarioMasta64** [comment]
only issue with this is, is it directly linked? if it is expect backlash. if its inputted then its basically just another pkgi.txt .3. either way i dont see much point.

after all you closed another issue for the fact that the user should be creating these lists, and in-turn they should also be able to know their own zRif's if they created it but you do you :p

**Comment by mmozeiko** [comment]
What do you mean by "directly linked"? pkgi won't do anything about creating license.db or downloading it. It will be up to user to create and manage it. I believe there will be new feature in next VitaShell versions, which will allow to populate this database with license you have. That means pkgi could link one public list of pkg files without zRIF strings. It could be shared in public, and I'll link it or even use by default in github. licenses.db file allows clean separation for NoNpDrm fake license files and pkg unpacking.

**Comment by MarioMasta64** [comment]
so basically it would look for zrifs that are already in the system? :D that sounds neato

**Comment by mmozeiko** [comment]
Yes, potentially. Such feature for VitaShell was discussed somewhere. Using licenses.db will be just a nice extra feature. It won't be mandatory to use it if you don't want to.

**Comment by MarioMasta64** [comment]
in this way you wouldnt even need lists and could just download what you own from your system. tho id imagine that would piss off tons of people if that was the only way to do it after all theyd want to play their "backups" too lol
